### ISSUE TITLE
Suggestion. Sound execution at the same time as the animation

### ISSUE THREAD

**Issue by Fridan99** [otigon/automated-jb2a-animations#135]
In the A-A, sound execution section, I don't know if it could be optimized. For example, in the spells that have a template (Misty Step, Fireball, etc...) the sound is not executed when you finish putting the template in the scene, but when you cast the spell. I know there is a delay option, but it is a bit random.... what may take some player 2 sec to put the template, to another may take 8, 9 sec... for these cases the sound is not executed at the same time that the animation appears. 

I don't know if for the cases where a template has to be entered, the sound can be executed when it has been staged.

![Sacreenshot 2](https://user-images.githubusercontent.com/86131373/125165815-83572a00-e190-11eb-9585-3849fe2665a4.jpg)
![Sacreenshot 3](https://user-images.githubusercontent.com/86131373/125165820-86eab100-e190-11eb-8f72-96ce466a8a90.jpg)

**Comment by otigon** [comment]
I'll be redoing alot of the sound options after I'm finished with this current rewrite of the module. The way things are handled now are a bit janky and difficult to customize. I'll get this fixed during that update

**Comment by Fridan99** [comment]
I have noticed another detail..... when putting the sound in a template, if the sound is very long and I delete the template, the sound continues to run until the end. 

I have noticed when using the Spirit Guardians, I have attached a sound of 01:32min, and when I remove the template, the sound continues. It would be nice if the sound system had 2 elements: 

1. Loop mode.... (this is good for the sounds of aura spells, or others that last a long time)
2. End the sound when the template is removed. 

Translated with www.DeepL.com/Translator (free version)
